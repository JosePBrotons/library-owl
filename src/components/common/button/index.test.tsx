import React from 'react';
import renderer from 'react-test-renderer';
import Button from '.';

test('renders button component correctly', () => {
    const props = {
        text: 'Sign In',
        disabled: false,
        onPress: jest.fn
    };
    const btnCmp = renderer.create(<Button {...props} />).toJSON();
    expect(btnCmp).toMatchSnapshot();
});
