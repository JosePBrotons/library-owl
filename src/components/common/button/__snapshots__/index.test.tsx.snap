// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders button component correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgba(57, 172, 233, 1)",
      "borderRadius": 20,
      "height": 40,
      "justifyContent": "center",
      "marginBottom": 10,
      "marginTop": 5,
      "opacity": 1,
      "width": "70%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "rgba(255, 255, 255, 1)",
        "fontFamily": "Nunito-Bold",
        "fontSize": 16,
        "lineHeight": 22,
      }
    }
  >
    Sign In
  </Text>
</View>
`;
